samples/bluetooth/mesh/light_lc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
I'd suggest renaming this sample to light_ctrl to match the API it's demonstrating.

Reply:
Done